cpu/stm32f7/vectors.c

```diff
@@ -0,0 +1,257 @@
+WEAK_DEFAULT void isr_dma2_stream4(void);
+WEAK_DEFAULT void isr_eth(void);
+WEAK_DEFAULT void isr_eth_wkup(void);
+WEAK_DEFAULT void isr_can2_tx(void);
+WEAK_DEFAULT void isr_can2_rx0(void);
+WEAK_DEFAULT void isr_can2_rx1(void);
+WEAK_DEFAULT void isr_can2_sce(void);
+WEAK_DEFAULT void isr_otg_fs(void);
+WEAK_DEFAULT void isr_dma2_stream5(void);
+WEAK_DEFAULT void isr_dma2_stream6(void);
+WEAK_DEFAULT void isr_dma2_stream7(void);
+WEAK_DEFAULT void isr_usart6(void);
+WEAK_DEFAULT void isr_i2c3_ev(void);
+WEAK_DEFAULT void isr_i2c3_er(void);
+WEAK_DEFAULT void isr_otg_hs_ep1_out(void);
+WEAK_DEFAULT void isr_otg_hs_ep1_in(void);
+WEAK_DEFAULT void isr_otg_hs_wkup(void);
+WEAK_DEFAULT void isr_otg_hs(void);
+WEAK_DEFAULT void isr_dcmi(void);
+WEAK_DEFAULT void isr_rng(void);
+WEAK_DEFAULT void isr_fpu(void);
+WEAK_DEFAULT void isr_uart7(void);
+WEAK_DEFAULT void isr_uart8(void);
+WEAK_DEFAULT void isr_spi4(void);
+WEAK_DEFAULT void isr_spi5(void);
+WEAK_DEFAULT void isr_spi6(void);
+WEAK_DEFAULT void isr_sai1(void);
+WEAK_DEFAULT void isr_ltdc(void);
+WEAK_DEFAULT void isr_ltdc_er(void);
+WEAK_DEFAULT void isr_dma2d(void);
+WEAK_DEFAULT void isr_sai2(void);
+WEAK_DEFAULT void isr_quadspi(void);
+WEAK_DEFAULT void isr_lptim1(void);
+WEAK_DEFAULT void isr_cec(void);
+WEAK_DEFAULT void isr_i2c4_ev(void);
+WEAK_DEFAULT void isr_i2c4_er(void);
+WEAK_DEFAULT void isr_spdif_rx(void);
+
+/* interrupt vector table */
+ISR_VECTORS const void *interrupt_vector[] = {
+    /* Exception stack pointer */
+    (void*) (&_estack),             /* pointer to the top of the stack */
+    /* Cortex-M7 handlers */
+    (void*) reset_handler_default,  /* entry point of the program */
+    (void*) nmi_default,            /* non maskable interrupt handler */
+    (void*) hard_fault_default,     /* hard fault exception */
+    (void*) mem_manage_default,     /* memory manage exception */
+    (void*) bus_fault_default,      /* bus fault exception */
+    (void*) usage_fault_default,    /* usage fault exception */
+    (void*) (0UL),                  /* Reserved */
+    (void*) (0UL),                  /* Reserved */
+    (void*) (0UL),                  /* Reserved */
+    (void*) (0UL),                  /* Reserved */
+    (void*) isr_svc,                /* system call interrupt, in RIOT used for
+                                     * switching into thread context on boot */
+    (void*) debug_mon_default,      /* debug monitor exception */
+    (void*) (0UL),                  /* Reserved */
+    (void*) isr_pendsv,             /* pendSV interrupt, in RIOT the actual
+                                     * context switching is happening here */
+    (void*) isr_systick,            /* SysTick interrupt, not used in RIOT */
+    /* STM specific peripheral handlers */
+    (void*) isr_wwdg,
+    (void*) isr_pvd,
+    (void*) isr_tamp_stamp,
+    (void*) isr_rtc_wkup,
+    (void*) isr_flash,
+    (void*) isr_rcc,
+    (void*) isr_exti0,
```

Comment:
It seems like every other stm32 define every `isr_exti*` as `isr_exti`, gpio interrupt are crashing otherwise.